### PR TITLE
Move bootstrax's context into contexts.py

### PR DESCRIPTION
**Pull request by JelleAalbers**
This:
  * Moves the online processing context created in bootstrax out to the main context file. This is mainly so you can load nT data in notebooks on the eb machine after `st = straxen.contexts.xenonnt_online()`.
  * Fixes some name/number handling in rundb.py.

I tested it briefly on eb0; bootstrax still seems to work.

### CODE REVIEW

File: bin/bootstrax [hunk 1]
```diff
@@ -222,19 +222,11 @@ def new_context():
...
-            runid_field='number',
-            new_data_path=output_folder,
-            mongo_dbname=dbname),
-        config=straxen.contexts.xnt_common_config,
-        **context_opts)
-    st.context_config['allow_multiprocess'] = True if args.cores > 1 else False
-    st.context_config['allow_shm'] = True if args.cores > 1 else False
-    st.context_config['max_messages'] = args.max_messages
-    return st
+    return straxen.contexts.xenonnt_online(
```
Comment (jorana) [inline bin/bootstrax]: If we don't specify the ```output_folder``` the data isn't going to a place where datamanager can fetch it.

**Comment by jorana** [comment]
Other than the minor comment it looks good Jelle!

File: bin/bootstrax (hunk 1 above)
Comment (JelleAalbers) [reply bin/bootstrax]: Good catch, thanks. Should be fixed now.

**Comment by jorana** [comment]
Thanks Jelle, I've removed some superfluous DB calls that were now obsolete (
76b02f2). Since these commands were now a little scattered around, I bundled them again.

Also changed the reference you mentioned last Thursday for the nT-simulator (f03fe85) as you are editing context.py anyway.

**Pull request merged**